sdks/Cordova.md

```diff
@@ -0,0 +1,136 @@
+# Apache Cordova AeroGear Services
+
+## Motivation
+
+Provide set of the SDK's that can be used for Cordova based development.
+Provide best performance and user experience on the platform.
+
+## Proposed solution 1
+
+Pure javascript solution for the mobile configuration and orchestration 
+that will allow cooperability between JavaScript and Native.
+Configuration parser and other pure JS code willnot require any native implementation and can be reused for cordova/react native implementations. Individual SDK will may by default be implemented in pure JavaScript or as Cordova/React plugins. 
+
+## Advantages of the solution 1
```

Comment:
> Significantly reduced effort to develop and maintain Cordova Support by reusing available components.
Significantly reduced compared to what?
Also what do you mean by 'reusing available components'? What components are being reused? 
Would that be the same as the point about 'Ability to reuse existing cordova plugins that perform native tasks'?